frontend/components/panels/VaultLiquidationPanel.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+        <TextBlock v-if="isExplanationsShown">
+            Successful liquidation of the vault will transfer combined incentive into the wallet that executes the
+            transaction (or a specified wallet). This transaction will start a collateral auction that one can also
+            participate in on a <nuxt-link :to="`/collateral?network=${network}`">separate page</nuxt-link>.
+        </TextBlock>
+        <div class="flex mt-4 justify-end gap-5">
+            <ExecuteWithOtherWalletModal
+                :is-shown.sync="isExecuteToAnotherWalletModalShown"
+                :default-wallet="walletAddress"
+                class="pb-3"
+                @execute="executeWithOtherWallet"
+                @close="closeExecuteToOtherWalletModal"
+            />
+            <BaseButton
+                :disabled="!isWalletConnected || !isLiquidatable || areLimitsMissing || areLimitsReached"
+                @click="isExecuteToAnotherWalletModalShown = true"
+            >
+                Liquidate to another wallet
+            </BaseButton>
+            <BaseButton
+                type="primary"
+                :disabled="!isWalletConnected || !isLiquidatable || areLimitsMissing || areLimitsReached"
+                :is-loading="isLiquidating"
+                @click="$emit('liquidate', walletAddress)"
+            >
+                Liquidate {{ collateralType }}:{{ auctionId }} vault
+            </BaseButton>
+        </div>
+    </BasePanel>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import BigNumber from 'bignumber.js';
+import ExecuteWithOtherWalletModal from '../modals/ExecuteWithOtherWalletModal.vue';
+import BaseButton from '~/components/common/inputs/BaseButton.vue';
+import TextBlock from '~/components/common/other/TextBlock.vue';
+import BasePanel from '~/components/common/other/BasePanel.vue';
+import { LiquidationLimits, VaultTransactionState } from '~/../core/src/types';
+
+export default Vue.extend({
+    name: 'VaultLiquidationPanel',
+    components: {
+        ExecuteWithOtherWalletModal,
+        BaseButton,
+        BasePanel,
+        TextBlock,
+    },
+    props: {
+        auctionId: {
```

Comment:
There are no `auctionId` here, we're liquidating a vault, not auction
```suggestion
        vaultId: {
```